src/core/packet.lua

```diff
@@ -94,6 +94,11 @@ end
 -- Return pointer to packet data.
 function data (p) return p.data end
 
+-- Return physical address of packet data
```

Comment:
I don't think this function justifies extending the packet API. [virtual_to_physical](https://github.com/SnabbCo/snabbswitch/tree/master/src#memory-corememory) is public, what's wrong with:

```
local memory = require("core.memory")

phys = memory.virtual_to_physical(packet.data(p))
```